app/models/user.rb

```
@@ -6,6 +6,7 @@ class User < ApplicationRecord
   has_many :sessions, dependent: :destroy
   has_many :rsvps
   has_many :photos
+  has_many :created_events, class_name: "Event", foreign_key: :creator, inverse_of: :creator
```

RuboCop (1.75.5)

[convention] Specify a `:dependent` option.

(Rails/HasManyOrHasOneDependent)